### ISSUE TITLE
link error

### ISSUE THREAD

**Issue by pmuvva** [CocoaPods/CocoaPods#3163]
Hi,

I have an existing iOSClient.xcodeproj  which will generate static library and I am trying to link with Pods but I am getting linking error.
My pod file:
xcodeproj 'iOSClient.xcodeproj', { 'DebugProd' => :debug, 'DebugTest' => :debug, 'ReleaseProd' => :release}
platform :ios, '8.0'

pod "Brightcove-Player-SDK"

I am getting below errors:

/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: -dynamic not specified the following flags are invalid: -ObjC 
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: can't locate file for: -lPods
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: file: -lPods is not an object file (not allowed in a library)

Please help me to resolve this issue.

**Comment by pmuvva** [comment]
While I am doing Pod Install I got below warnings:

[!] The `iOSClient [Debug]` target overrides the `OTHER_LDFLAGS` build setting defined in `Pods/Target Support Files/Pods/Pods.debug.xcconfig'. This can lead to problems with the CocoaPods installation
    - Use the `$(inherited)` flag, or
    - Remove the build settings from the target.

[!] The `iOSClient [Release]` target overrides the `OTHER_LDFLAGS` build setting defined in `Pods/Target Support Files/Pods/Pods.release.xcconfig'. This can lead to problems with the CocoaPods installation
    - Use the `$(inherited)` flag, or
    - Remove the build settings from the target.


I updated "inherited" flag but still I am getting same linking error.
